spec/rails_applications/7.1.3/.dockerignore

```diff
@@ -0,0 +1,37 @@
+# See https://docs.docker.com/engine/reference/builder/#dockerignore-file for more about ignoring files.
+
+# Ignore git directory.
+/.git/
+
+# Ignore bundler config.
+/.bundle
+
+# Ignore all environment files (except templates).
+/.env*
+!/.env*.erb
+
+# Ignore all default key files.
+/config/master.key
+/config/credentials/*.key
+
+# Ignore all logfiles and tempfiles.
+/log/*
+/tmp/*
+!/log/.keep
+!/tmp/.keep
+
+# Ignore pidfiles, but keep the directory.
+/tmp/pids/*
+!/tmp/pids/.keep
+
+# Ignore storage (uploaded files in development and any SQLite databases).
+/storage/*
+!/storage/.keep
```

Comment:
You've mentioned above what you're keeping but not for all sections. `!/storage/.keep` means you're keeping this directory correct? Would be good to add in the comment in that case plus for the others :)

Reply:
Hi @sumitramanga this is just part of the standard rails layout - so I wouldn't want to make any changes here as this is upstream from Rails (see https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/dockerignore.tt#L30)

Reply:
No worries! Thank you for the context